contributing.md

```diff
@@ -31,6 +31,9 @@ The repository is structured as follows:
 
 - `./module_utils/network_lsr/` - Contains other files that are useful for the network
   role (e.g. the YAML argument validator)
+ 
+- `./module_utils/network_nm_lsr/` - Contains other files that are useful for the network
```

Comment:
It seems to me to be a lot less complex to use `./module_utils/network_lsr/nm_provider/` or some other subdirectory of `network_lsr`. Then you don't need to duplicate the extra work to get the path added to multiple places. What do you think?

Reply:
Ansible 2.8 does not allow folder in `network_lsr`.

Reply:
> Ansible 2.8 does not allow folder in `network_lsr`.

How so?  I'll note that when I install ansible 2.8 with pip, the module_utils directories have sub-directories e.g. `module_utils/storage/emc/` - I don't know if this is "allowed" or if you can do `import ansible.module_utils.storage.emc.emc_vnx` and it "works", but it is there in the ansible 2.8 package.

Reply:
I tried on RHEL 8 using ansible 2.8. The `explode` command shows the sub-folder is not send to remote host.

Reply:
Yeah. with `ansible.module_utils.storage.emc.emc_vnx`, the sub-folder works. Let me ammend this PR.

Reply:
Thanks for the hint. New patch uploaded.